react-ui/src/components/Layouts/Home.js

```diff
@@ -1,13 +1,16 @@
 import React, { Component, Fragment } from "react";
+
 import Title from "../Title"
+import AppointmentList from "../AppointmentList/AppointmentList";
 
-class IndexDashboard extends Component {
+class Home extends Component {
```

Comment:
Renamed this page to Home to increase clarity as IndexDashboard may be confusing.

Reply:
Sounds good to me.